cmd/quickstart/quickstart.go

```diff
@@ -0,0 +1,32 @@
+package quickstart
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd() *cobra.Command {
+	var quickstartcmd = cobra.Command{
+		Use:   "quickstart [sdk]",
+		Short: "Create a minimal getting started app for the specified SDK",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			sdk := args[0]
+			switch strings.ToLower(sdk) {
+			case "ts", "typescript":
+				printInstructionsForTypeScript()
+			default:
+				fmt.Printf("Error: Unknown SDK '%s'.\n", sdk)
```

Comment:
also look at the complete promise cmd for example. 

use cmd.* for io and distinguish between printf and printerr. this should be err. 

we do this to follow dependency injection principle.